docs/readmewidget.md

```diff
@@ -0,0 +1,167 @@
+# Onramper
+
+[Instructions of how to integrate the widget in your website here.](docs.onramper.dev)
+
+This project is created using create-react-app with typescript, here you will find some notes about how to customize or build the widget from the source files. In this repo you will find the source files of Onramper's buy page (`https://widget.onramper.com`) and the source files of the widget component (`https://npm.org/package/@onramper/widget`)
+
+Directory structure (only main files listed):  
+
+```
+.
+├── dist # generated after build
+├── ...
+├── src
+│   ├── OnramperWidget
+│   ├── ...
+│   ├── App.tsx
+│   ├── index.css
+│   └── index.tsx 
+├── ...
+├── component.tsconfig.json
+├── package.json
+├── tsconfig.json
+├── webpack.config.js
+└── index.html
+```
+
+| File / folder                                             | Description                                                                   |
+| --------------------------------------------------------- | ----------------------------------------------------------------------------- |
+| `./src/OnramperWidget`                                    | Source files of the widget component. [See more](#onramperwidget).            |
+| `./component.tsconfig.json`<br>`./webpack.config.js`      | Config files used to transpile and bundle the widget component.               |
+| `./src/App.tsx`<br>`./src/index.css`<br>`./src/index.tsx` | Buy page files.                                                               |
+| `./dist`                                                  | Bundled component.                                                            |
+| `./tsconfig.json`                                         | Typescript configuration file used by `react-scripts` app.                    |
+| `./package.json`                                          | Describes scripts for develop, test and build the component and the buy page. |
+
+## Setup
+Install dependencies:
+```shell
+npm install
+```
+
+## Developement
+Start a developement server:
+```
+npm start
+```
+
+## Build
+Build the buy page as an static site into the `./build` folder:
+
+```shell
+npm run build
+```
+
+Build the widget component source files into the `./dist` folder:
+
+```shell
+npm run build:component
+```
+
+## OnramperWidget
+Directory structure (only main files listed):  
+
+```
+OnramperWidget
+├── NavContext
+├── ApiContext
+│   ├── ...
+│   └── api 
+├── ...
+├── steps
+│   ├── ...
+│   └── Step 
+└── index.tsx
+```
+
+| File / folder      | Description                                                                                                                                                       |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I added a new folder (which handles moonpay API), should we document it? On another note, should we move that folder somewhere else and/or publish it as a package?